acquisitions/src/main/resources/thunderjet/mod-ebsconet/features/cancel-order-lines-with-ebsconet.feature

```diff
@@ -7,40 +7,40 @@ Feature: Cancel order lines with ebsconet
 
     * url baseUrl
 
-    * callonce login testAdmin
+    * call login testAdmin
```

Comment:
Here is reason why callonce changed to call:

lets say we have fundId one fundId for one db. If I want to create second fundId it will throw error.
Callonce call only time and generate uuid for example only one time. 
in Performance testing we use different users and repetable cycle with 10 times. So If I generate one uuid for all 10 tests it will throw in second cycle. 
 
For login it will genreate different token for different user everytime in every test. 
 
For different users we need different token and different uuids.